Profile_App/form/delete.php

```diff
@@ -0,0 +1,36 @@
+<?php
+
+/**
+ * Delete employee data file.
+ */
+
+require_once('config/database_config.php');
+require_once('config/constants.php');
+
+if (isset($_GET['emp_id']) && preg_match('/^[0-9]*$/', $_GET['emp_id'])) {
+    $emp_id = stripslashes($_GET['emp_id']);
```

Comment:
Use mysqli_real_escape_string function instead of stripslashes
http://php.net/manual/en/mysqli.real-escape-string.php